src/java.base/share/classes/java/util/jar/JarInputStream.java

```diff
@@ -103,11 +145,13 @@ private JarEntry checkManifest(JarEntry e)
     }
 
     /**
-     * Returns the {@code Manifest} for this JAR file, or
-     * {@code null} if none.
+     * Returns the {@code Manifest} for this JAR file when it is the first entry
+     * in the stream (or the second entry the first entry in the stream is
+     * {@code META-INF/} and the second entry {@code META-INF/MANIFEST.MF}), or
```

Comment:
Missing "is": "and the second entry is ..."

Reply:
fixed